Fix incorrect mouse position example

trevordevore opened the pull request:
The <mouse position> example used <click position> instead of <mouse position>.

trevordevore committed:
Fix incorrect mouse position example

The <mouse position> example used <click position> instead of <mouse position>.

trevordevore committed:
Updated references to clicks and the example

* This doc entry referenced mouse click in a couple of places but it returns the mouse location. Those references were fixed.
* Updated the summary to mention that location is relative to top left of the widget.

peter-b commented:
@trevordevore It looks to me like you've accidentally changed the documentation for `the click position` instead of the documentation for `the mouse position`!

trevordevore commented:
Right you are. No wonder there were so many references to the mouse click :-)

peter-b commented:
@livecode-vulcan review not ok 0a4c439

trevordevore committed:
Fixes the example in the proper location

Previous commits modified the <mouse click> entry rather than the <mouse position> entry.

trevordevore committed:
Fixed a typo in <click position>

trevordevore commented:
Okay. All better now.

livecode-vulcan commented:
💔 review by @peter-b not ok 0a4c439a3537e7adc9017394d67b89691b6c5716

peter-b commented:
@trevordevore I haven't done much stuff with the UI features in LCB. When you say, "relative to the widget", does that mean that `the mouse position` is `[0,0]` when the mouse is pointing at the top left corner of the widget's rect?

trevordevore commented:
@peter-b Yes it does. In LCS the mouseLoc() function is going to return the coordinates relative to the stack, not the control. So there is a chance of an incorrect assumption being made by the LCB developer coming from LCS as they may not be thinking in terms of the widget having it's own coordinate system.

peter-b commented:
@trevordevore That makes sense. Thank you for the clarification!

@livecode-vulcan review ok 79691b5

livecode-vulcan commented:
💙 review by @peter-b ok 79691b5c948dcde20e6994b93425bff4f2b133d0

livecode-vulcan commented:
😎 test success 79691b5c948dcde20e6994b93425bff4f2b133d0
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1167)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/1178)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1164)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1171)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/1182)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/1184)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/1186)
